ratatui-core/src/layout/direction.rs

```diff
@@ -17,6 +17,17 @@ pub enum Direction {
     Vertical,
 }
 
+impl Direction {
+    #[inline]
+    #[must_use]
+    pub const fn other(self) -> Self {
+        match self {
+            Self::Horizontal => Self::Vertical,
+            Self::Vertical => Self::Horizontal,
+        }
+    }
+}
+
```

Comment:
This seems useful regardless of the new layout. How about submitting this change as a small PR?